insightly.py

```diff
@@ -264,6 +264,8 @@ def __init__(self, apikey='', version='2.2', dev=None, gzip=True, debug=True, te
                     self.owner_id = u.get('USER_ID', None)
                     self.owner_name = u.get('FIRST_NAME','') + ' ' + u.get('LAST_NAME','')
                     self.contact_id = u.get('CONTACT_ID', None)
+                    self.email_dropbox = '%s-%s@mailbox.insight.ly' % (u.get('FIRST_NAME', '').lower(),
```

Comment:
We added a new field to the api response object called EMAIL_DROPBOX_ADDRESS, should have it in production within a couple of days. If we could just map that new field to self.email_dropbox that would be great!